### DISCUSSION TITLE
How to add prefix to all nodes?

### DISCUSSION THREAD

**Discussion by jaekunchoi** [ueberdosis/tiptap#4642] [General Questions]
I want to add p, h1, h2, h3, h4, h5 etc to start of each line as prefix for each node.

I tried using nodeview but not really how we can do this. Is there a way to achieve this?

**Comment by gethari** [comment]
maybe you can programatically do before setting your actual content or when the editor instance is `created`

```ts
insertParagraph()
headings.forEach(heading=>  editor.toggleHeading(heading))
```

**Reply by jaekunchoi** [reply]
Is there a way to do this with extension globally for all nodes?
